.travis.yml

```diff
@@ -0,0 +1,26 @@
+
+sudo: required
+dist: xenial
+language: python2.7
+
+services:
+  - docker
+
+branches:
+  only:
+    - master
+
+before_install:
+  - sudo apt-get update
+  - curl https://storage.googleapis.com/git-repo-downloads/repo > ~/bin/repo
+  - chmod a+x ~/bin/repo
+  - git config --global color.ui false
+
+script:
+  - repo init -u .
+  - cp default.xml .repo/manifests/
+  - repo sync
+  - repo forall -p -c 'git checkout $REPO_RREV'
+  - cd prplMesh/tools/docker/builder
```

Comment:
Same here: all of the above is already done inside the docker image. And Travis CI will check out the current repo into the current directory. So if we put this .travis.yml in the prplMesh repo, it is enough to do
```
sudo docker image build --tag prplmesh-build tools/docker/builder
```